--- a/modal_aero_wateruptake.F90
+++ b/modal_aero_wateruptake.F90
@@ -27,6 +27,8 @@
    modal_aero_wateruptake_sub
 
 public :: modal_aero_wateruptake_reg
+
+public :: makoh_cubic, makoh_quartic, modal_aero_kohler
 
 real(r8), parameter :: third  = 1._r8/3._r8
 real(r8), parameter :: pi43   = pi*4.0_r8/3.0_r8
--- a/modal_aero_newnuc.F90
+++ b/modal_aero_newnuc.F90
@@ -23,7 +23,7 @@
 
 ! !PUBLIC MEMBER FUNCTIONS:
   public :: modal_aero_newnuc_sub, modal_aero_newnuc_init, &
-            mer07_veh02_nuc_mosaic_1box
+            mer07_veh02_nuc_mosaic_1box, binary_nuc_vehk2002, pbl_nuc_wang2008
 
 
 ! !PUBLIC DATA MEMBERS:
--- a/modal_aero_coag.F90
+++ b/modal_aero_coag.F90
@@ -19,7 +19,7 @@
   private
   save
 
 ! !PUBLIC MEMBER FUNCTIONS:
-  public :: modal_aero_coag_sub, modal_aero_coag_init, getcoags_wrapper_f
+  public :: modal_aero_coag_sub, modal_aero_coag_init, getcoags_wrapper_f, getcoags
 
 ! !PUBLIC DATA MEMBERS:
